styles/components/nav.css

```diff
@@ -0,0 +1,25 @@
+.nav {
+  background-color: var(--orange);
+  width: 324;
```

Comment:
i believe, here you wanted to add `px`, but that wouldn't be responsive.

Reply:
haha thanks, you are right, this was the thought in first place, but again yes, i should make it responsive😊
Thank you for the quick review Tanja! 🤩